Carlos-Reyna/react-apprenticeship-capstone4
# chore: added all the necessesary code to complete capstone delivery 3…

## Description
```
< Replace with the PR description >

-Added Product Search
-Added Product Detail
-Update Home page to fetch items from Prismic
-Update Product list to fetch items from Prismic
-Update General App Layout
-Added Routing
-Remove unnecessary styles

```
## Type of Change


## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

- Download or fork this repository
- Go to the project folder
- Install all the necessary dependencies with yarn
- Use the following command to run the app: yarn start
- Navigate to localhost:3000/products
- Click on a displayed item name link
- Verify that the product detail page is displayed
```

## What is the deliverable?
```
React Apprenticeship deliverable 3
```
## Screenshots and links
```
You can see the deployment preview at: https://elegant-lamarr-a24cc4.netlify.app/
```
## Notes
```
Had to add this exceptions to eslint:     'jsx-a11y/click-events-have-key-events': 'off',
    'jsx-a11y/no-noninteractive-element-interactions': 'off',

There was a conflict with the carrousel component, since it is a third party library it needs a an image to work properly, the "correct" way was to display a an input with an image as a background, but when the input was used, the library suddenly stopped working. 
```
## Checklist
